run-tests.sh

```diff
@@ -1,12 +1,47 @@
-#!/bin/sh
+#!/bin/bash
+#
+# This file is part of REANA.
+# Copyright (C) 2020 CERN.
+#
+# REANA is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
 
 # Quit on errors
 set -o errexit
 
 # Quit on unbound symbols
 set -o nounset
 
-echo '==> [INFO] Checking for broken links...'
-awesome_bot --allow-dupe --skip-save-results --allow-redirect --white-list https://reana.cern.ch **/*.html
-echo '==> [INFO] Testing Docker image build...'
-docker build -t reanahub/wwwreanaio .
+check_script () {
+    shellcheck run-tests.sh
+}
+
+check_docstyle () {
+    shopt -s globstar
```

Comment:
WDYT about moving it next to the `set` ones and add a comment "Enable globstar (**)"?